### DISCUSSION TITLE
Beginner basics for SQLStudio + QTScript

### DISCUSSION THREAD

**Discussion by DBJDBJ** [pawelsalawa/sqlitestudio#3448] [Q&A]
Hi, great product. Please advise where can I read about total beginner basics for QTScript-ing?
I think I am doing it fine but for some reason, it does not work. I never used SQLiteStudio before. I am trying to make this work:

SELECT word from words where (word == reverse(word))

where reverse() is a QTScript function, [seen on the screen dump](https://snag.gy/uy8ZWG.jpg)

I am getting:

[05:24:07] Error while executing SQL query on database 'EN_DICTIONARY': no such function: reverse

SQLStudio version is 3.2.1, OS is WIN10 PRO
Please advise,
Thanks ...

**Comment by pawelsalawa** [comment]
You named your function "function reverse".

Just remove "function " prefix out of the name, keep just "reverse".

**Comment by DBJDBJ** [comment]
And it worketh. Grateful as I am I have to say I still have not been able to find an online text which will also ease the burden on you to answer this kind of questions.

For example: if a user clicks on green '+' on a functions editing dialogue, the app puts the word 'function' in the 'Function name' field. Without knowing what I am doing I just added a function name thus producing a typical beginners mistake.  Perhaps if you would leave that field empty the number of beginners complaining will be much smaller with the proper introductory text too.

Great App. Kind regards ...

**Comment by pawelsalawa** [comment]
You are absolutely right. This is confusing. I'll change it.

**Comment by DBJDBJ** [comment]
Glad to be of service ...

**Comment by DBJDBJ** [comment]
Ah, sorry but more ideas are coming in  :)  How about this? 

User clicks on the green '+' and app creates some kind of complete "hello world" script? 
With perhaps some useful comments in the code, links and a such?

I might think that will ease the questions about functions, considerably?

Kind regards ...

**Comment by pawelsalawa** [comment]
Good idea. I will incorporate it somehow.
